### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Just a little trouble: seems like local storage data aren't saved. Do you know about this?

Also, here's a screenshot about the UI changes:

screenshot 2018-10-31 at 00 19 58

### CODE REVIEW

**Comment by Joacchim** [comment]
Well, there's actually no data saved yet, since the register process isn't completely implemented (we have a part of the UI but that's it, storage is not yet linked to the UI - partly because we want to let the user choose the label when registering a comic/manga anyways)

File: web-extensions/chrome/engine/storage.js [hunk 1]
```diff
@@ -130,12 +130,18 @@ Storage.prototype.get = function(keys, cb) {
  *
  */
 Storage.prototype.set = function(dataset, cb) {
-    console.log('Wrapper.set');
+    console.log(`Wrapper.set "${JSON.stringify(dataset)}"`);
```
Comment (Joacchim) [inline web-extensions/chrome/engine/storage.js]: that will become huge very quickly (with the "mapping" which contains all sources)

File: web-extensions/chrome/engine/datamodel.js [hunk 2]
```diff
@@ -246,6 +246,21 @@ BmcComic.prototype.serialize = function() {
     };
 }
 
+/**
+ * This method loops over sources.
+ *
+ * @param {object} callback - The callback to be called on each source.
+ *
+ */
+BmcComic.prototype.iterSources = function(callback) {
```
Comment (Joacchim) [inline web-extensions/chrome/engine/datamodel.js]: Thought I had added it Oo ?! Looks like I somehow forgot to add it despite that... my bad !

**Comment by GuillaumeGomez** [comment]
> Well, there's actually no data saved yet, since the register process isn't completely implemented (we have a part of the UI but that's it, storage is not yet linked to the UI - partly because we want to let the user choose the label when registering a comic/manga anyways)

One click away is very annoying. Even more in case of testing.

**Comment by GuillaumeGomez** [comment]
It's useful because you expect global page's elements state to be the same across the different pages.

**Pull request merged**